server/kafka/kafkaproducer.go

```diff
@@ -0,0 +1,69 @@
+package kafka
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/cosminrentea/gobbler/server/configstring"
+	"github.com/cosminrentea/gobbler/server/service"
+	"time"
+)
+
+type Producer interface {
+	service.Startable
+	service.Stopable
+	Report(topic string, bytes []byte, key string)
```

Comment:
This would allow a streaming mechanism any time :dagger: For example you can pass a json.Encoder which accepts an io.writer. And this way you can connect the writer to a reader and encode the data and save memory. All this is theory btw :D not completely sure